### PR TITLE
Update Github issue template and documentation with the new 3.1 option to "Create Debug Archive".

### PR DESCRIPTION
**Pull request by camilasan**
The doc should (more or less) look like this:

docs

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -14,9 +10,9 @@ Guidelines for submitting issues:
     - You can also filter by appending e. g. "state:open" to the search string.
     - More info on search syntax within github: https://help.github.com/articles/searching-issues
     
-* Please fill in as much of the template below as possible. We know it is a pain sometimes, but especially without logs there is often not much we can do: really, if we would have seen the issue you encoutered before, we would already have fixed it. So we did NOT see it, and we will need YOUR help to find out what is wrong and fix it. The logs are absoutely crucial for that. Expect us to quickly close issues without logs or other information we need. If you don't have time to gather the required information, we don't either.
+* Please fill in as much of the template below as possible. The logs are absoutely crucial for the developers to be able to help you. Expect us to quickly close issues without logs or other information we need.
```
Comment (er-vin) [inline .github/issue_template.md:13]: typo: absolutely

File: .github/issue_template.md [hunk 2]
```diff
@@ -70,15 +63,15 @@ Nextcloud version:
...
 
 Please use Gist (https://gist.github.com/) or a similar code paster for longer
 logs.
 
 1. Client logfile: 
-<!-- desktop client logs are a hard requirement for bug reports because we don't know how to do magic here :) -->
-Output of `nextcloud --logdebug --logwindow` or `nextcloud --logdebug --logfile log.txt`
-(On Windows using `cmd.exe`, you might need to first `cd` into the Nextcloud directory)
-(See also https://docs.nextcloud.com/desktop/2.3/troubleshooting.html#log-files)
+Under the "General" settings, you can click on "Create Debug Archive ..." to pick the location of where the desktop client will export the logs and the database to a zip file.
```
Comment (er-vin) [inline .github/issue_template.md:72]: Maybe mention this is for 3.1 and superior, and that for older releases the command line flags are the way to do?

Comment (camilasan) [reply .github/issue_template.md]: I fixed it. Is that clear now?

**Review by er-vin — APPROVED** [review]

File: .github/issue_template.md (hunk 2 above)
Comment (er-vin) [reply .github/issue_template.md]: Yes. Thanks.

**Pull request merged**